receiver/opencensus/opencensus.go

```diff
@@ -89,8 +93,14 @@ func (ocr *ocReceiver) StartTraceReception(ctx context.Context, ts spansink.Sink
 }
 
 func (ocr *ocReceiver) StartMetricsReception(ctx context.Context, ms metricsink.Sink) error {
-	// Currently a noop: TODO(@odeke-em, @songya): add the metrics receiver here
-	return nil
+	var err = errAlreadyStarted
+
+	ocr.startMetricsReceiverOnce.Do(func() {
+		ocr.metricsReceiver, err = ocmetrics.New(ms)
```

Comment:
Please check if err == nil i.e.
```go
if err == nil {
   srv := ocr.grpcServer()
   agentmetricspb.RegisterMetricsServiceServer(srv, ocr.metricsReceiver)
}
```
and we can do the same for octrace

Reply:
SG, added.